Sources/View/ReusableView.swift

```diff
@@ -20,15 +20,19 @@ public extension ReusableView where Self: UITableViewCell {
     init() {
         self.init(style: .default, reuseIdentifier: Self.reuseIdentifier)
 
-        setupLayout()
+        setupSubviews()
```

Comment:
Hmm... personally I too prefer setup, even though setUp is grammatically correct. However, I've just done a quick check to see if Apple use a particular way. `XCTest` uses `setUp`, so maybe we should change to that for consistency?

https://developer.apple.com/reference/xctest/xctestcase/1496262-setup